.github/workflows/main.yml

```diff
@@ -64,8 +64,8 @@ jobs:
                   name: playwright-test-results
                   path: test-results/
     release:
-        name: Release
-        if: github.ref == 'refs/heads/main'
+        name: ${{ github.ref == 'refs/heads/main' && 'Release' || 'Release Beta'}}
```

Comment:
It doesn't look like this work as expected @ttaylor-stack. See this screenshot from the UI checks in the PR
<img width="838" height="103" alt="Screenshot 2025-08-27 at 10 07 47" src="https://github.com/user-attachments/assets/9ed2b7ca-9e65-47c4-a967-893c2b5f9034" />